src/mtt_allocator.c

```diff
@@ -384,6 +384,19 @@ MEMKIND_EXPORT void *mtt_allocator_malloc(MTTAllocator *mtt_allocator,
     return ret;
 }
 
+MEMKIND_EXPORT void *mtt_allocator_calloc(MTTAllocator *mtt_allocator,
+                                          size_t num, size_t size)
+{
+    size_t alloc_size = num * size;
+    void *ret = mtt_allocator_malloc(mtt_allocator, alloc_size);
+    if (ret == NULL) {
+        log_fatal("Allocation failed!");
+        abort();
```

Comment:
I'm not sure if we should abort here - if mtt_allocator_malloc did not abort, why should we ? We can just propagate the null value, I wouldn't even add additional logs here.